### DISCUSSION TITLE
Default fonts dont seem to be loading correctly

### DISCUSSION THREAD

**Discussion by alex-r89** [mantinedev/mantine#4136] [Q&A]
Hi,

I like the default Mantine font, `Greycliff CF, sans-serif` but for some reason the default font is not working for me.

I am using nextjs, and have this set up in my `_app.jsx` file:

```js
export default function App() {

  return (
    <MantineProvider
       withGlobalStyles
       withNormalizeCSS>
       <Component {...pageProps} />
       </MantineProvider>
  )
}
```
I do not see the Greycliff CF font anywhere in the app with this setup. I have also tried specifying the font explicitly here too, like this:
```js
<MantineProvider
theme={{
fontFamily: 'Greycliff CF, sans-serif',
headings: { fontFamily: 'Greycliff CF, sans-serif' }
}}
withGlobalStyles
withNormalizeCSS>
```

This also did not work. To note, I also tried this, which also did not work:

```js
```js
<MantineProvider
theme={{
fontFamily: (theme) => `Greycliff CF, ${theme.fontFamily}`
headings: {
  fontFamily: (theme) => `Greycliff CF, ${theme.fontFamily}`
}
}}
withGlobalStyles
withNormalizeCSS>
```

**Strangely however**, I am able to get the Greycliff CF font working in individual components by using the `createStyles`, like so:

```js
const useStyles = createStyles((theme) => ({
  title: {
    fontFamily: `Greycliff CF, ${theme.fontFamily}`,
  },
}))

// then use this in said component as follows:
<Title className={classes.title}>Some text here</Title>
// This works and this Title has the Greycliff CF font
```

The issue here is that I need to to this in every single component in order to get the Greycliff CF font working.

Am I doing something wrong in the _app.jsx file?

Thanks,

**Comment by yverby** [comment]
You may need to add the font using `Global` component
https://mantine.dev/theming/typography/#load-custom-fonts

**Comment by rtivital** [comment]
Greycliff CF is a custom font, not default. You need to purchase a license and download it into your project if you want to use it, otherwise use other font.

**Reply by alex-r89** [reply]
> Greycliff CF is a custom font, not default. You need to purchase a license and download it into your project if you want to use it, otherwise use other font.

Thanks for the info. I wonder how am I seeing it on my site then?

**Reply by rtivital** [reply]
Most likely you are seeing a fallback font or you have this font installed locally

**Reply by alex-r89** [reply]
Ah, strange. I don't have it installed - I have not purchased a license/downloaded it. It must be a fallback, which seems odd as its not a default font as you say.
